### DISCUSSION TITLE
New Feature : Interactive Video : Knowledge 5.0

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Ed/Community#56] [Feature request]
https://project-sunbird.atlassian.net/browse/SB-30516

is merged with knowledge and indented to be part of R5.0

Feature Contribution Initiated by Avanti Fellows 

Wanted to update and notify this feature to be included for Sunbird Ed 5.0

CC @vinukumar-vs @coolbung @HarishGangula @karn-nahata

**Comment by sharathkashyap** [comment]
@baljeetsingh Thank you for the contribution of the feature. Could you please help the team with identified test cases and publish the results of test execution of the feature.

cc: @luckynvdu

**Reply by baljeetsingh** [reply]
@sharathkashyap @luckynvdu testcases have been attached to SB-30516 have been verified using sunbird ed <<-->> vidyadan

**Comment by karn-nahata** [comment]
@luckynvdu - just wanted to confirm that this will be picked in the next build cycle (5.0) for Sunbird ED

**Reply by baljeetsingh** [reply]
@karn-nahata next as in you are referring to 5.0 right ?

**Reply by karn-nahata** [reply]
Yes @baljeetsingh

**Reply by sharathkashyap** [reply]
@karn-nahata @baljeetsingh Kindly share the details of the artifacts of the delivery as part of 5.0 release cycle
cc: @luckynvdu
